### PR TITLE
Add Ahmed Rowaihi to the Data List

### PR DESCRIPTION
**Pull request by ahmedrowaihi**
## Changes proposed

- Ahmed Rowaihi to the Data List

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

I added a ```ahmedrowaihi.json``` data file : D

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4825"></a>

### CODE REVIEW

File: data/ahmedrowaihi.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      "icon": "FaComment"
+    },
+    {
+      "group": "Community",
+      "name": "SudoGroup: Join our Discord community",
+      "url": "https://discord.com/invite/td9WGydjxM",
+      "icon": "FaDiscord"
+    }
+  ],
+  "milestones": []
```
Comment (Panquesito7) [inline data/ahmedrowaihi.json:57]: There's no need in having this field if it's not being used. 🙂
```suggestion
```

**Review by himanshu1221 — APPROVED** [review]
**LGTM:+1:**

**Comment by ahmedrowaihi** [comment]
@Panquesito7 👍 all is good

**Review by Ananya2001-an — APPROVED** [review]
looks good.

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**